test/tests_for_sdkmr/EE2Runjob_test.py

```diff
@@ -230,27 +293,59 @@ def test_run_as_admin():
     # already a very large test. This may be something to be added later when needed.
 
     # Set up call returns. These calls are in the order they occur in the code
-    reqs = ResolvedRequirements(
-        cpus=cpus, memory_MB=mem, disk_GB=disk, client_group=client_group
-    )
+    jrr.normalize_job_reqs.return_value = {
+        "request_cpus": cpus,
+        "request_memory": mem,
+        "request_disk": disk,
+        "client_group": client_group,
+        "client_group_regex": True,
+        "debug_mode": True,
+    }
+    jrr.get_requirements_type.return_value = RequirementsType.BILLING
+    req_args = {
+        "cpus": cpus,
+        "memory_MB": mem,
+        "disk_GB": disk,
+        "client_group": client_group,
```

Comment:
scrolling through this file, there are a lot of places where dictionaries with these four args are created -- if I were writing this code, I'd either set up a default with every field populated and then create a new dict by merging that with whatever field you're changing, or have a dict with a subset of the fields that don't change, and merge it with extra params for the other fields. In either situation, you'd be creating a copy of the original dictionary before merging with the other params.

Reply:
Fixed